src/pages/AdminAdvancedSettings/AdminAdvancedSettings.tsx

```diff
@@ -77,26 +79,28 @@ export const AdminAdvancedSettings: React.FC = () => {
 
   return (
     <AdminRestricted>
-      <div className="AdminAdvancedSettings">
-        <div className="AdminAdvancedSettings__options">
-          {renderAdminAdvancedTabs}
+      <WithNavigationBar hasBackButton={false} withSchedule={false}>
```

Comment:
`WithNavigationBar` outside, `AdminRestricted` inside, navigation shouldn't be restricted only to admins

Reply:
Fixed in: ea4802ee1338ca4c0bf535218a1562a1d7c45bf3